### PR TITLE
v1.2: Update UGDG for Ingredient Management

### PR DESCRIPTION
**Pull request by rebstan97**
**Developer Guide:**
- Document implementation of delete ingredient feature (Implementation of other ingredient features to be added along the way)
- Update use cases

**User Guide:** 
- Add command aliases
- Change `list-ingredients-low` to `low-stock`

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -340,6 +340,43 @@ image::UndoRedoActivityDiagram.png[width="650"]
...
+* `DeleteIngredientCommand#equals()` – Checks if two `DeleteIngredientCommand` instances have the same attributes
+
+`DeleteIngredientByIndexCommand` and `DeleteIngredientByNameCommand` extends `DeleteIngredientCommand` and implement
+their own behaviour for these methods.
+
+The following sequence diagrams illustrate how the delete operation works.
+
+* Diagram 1: When user enters `delete-ingredient 1`
+
+image::DeleteIngByIndexSequenceDiagram.png[width="1000"]
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: Should just name it in full `DeleteIngredient...`. I got confused for a moment thinking it's `DeletIng`.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -793,7 +830,17 @@ Use case ends.
...
++
+Use case resumes at step 3.
+
+[none]
+* 2b. The user enters an invalid command or invalid command format.
+[none]
+** 2b1. App returns a message telling user that the command or command format is invalid.
+** 2b2. User requests to add ingredient again.
++
+Steps 2b1-2b2 are repeated until an ingredient name which does not exist is used.
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: This is confusing, I don't get what this statement means.

Comment (rebstan97) [reply docs/DeveloperGuide.adoc]: Thanks for pointing out, realised a number of typos. Have changed them

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (rebstan97) [reply docs/DeveloperGuide.adoc]: Changed

Comment (AZhiKai) [reply docs/DeveloperGuide.adoc]: Okay, please make a new PR for this.

**Pull request merged**